crates/graphql/src/schema.rs

```diff
@@ -580,6 +624,36 @@ impl BatchFn<String, Vec<NftCreator>> for NftCreatorBatcher {
     }
 }
 
+struct NftAttributeBatcher {
+    db_pool: Arc<Pool>,
+}
+
+#[async_trait]
+impl BatchFn<String, Vec<NftAttribute>> for NftAttributeBatcher {
+    async fn load(&mut self, addresses: &[String]) -> HashMap<String, Vec<NftAttribute>> {
+        let conn = self.db_pool.get().unwrap();
+        let mut hash_map = HashMap::new();
+
+        for address in addresses {
+            hash_map.insert(address.clone(), Vec::new());
+        }
+
+        let rows: Vec<models::MetadataAttribute> = attributes::table
+            .filter(attributes::metadata_address.eq(any(addresses)))
+            .load(&conn)
+            .unwrap();
+
+        rows.into_iter().map(Into::into).map(|attr: NftAttribute| {
```

Comment:
The clippy warnings here hint at the underlying issue - you have to call a method that consumes this iterator, otherwise it never runs as Rust executes iterators lazily